python/pyiceberg/cli/output.py

```diff
@@ -119,25 +124,40 @@ def describe_table(self, table: Table):
         output_table.add_row("Properties", table_properties)
         Console().print(output_table)
 
-    def describe_properties(self, properties: Properties):
+    def files(self, table: Table, io: FileIO) -> None:
+        metadata = table.metadata
+
+        snapshot_tree = Tree(f"Snapshots: {'.'.join(table.identifier)}")
+        for snapshot in metadata.snapshots:
```

Comment:
Am I reading this correctly that listing files will list all files for all snapshots? Would it be better if it listed the files for the current snapshot only and maybe we can have a flag to include files from all historical snapshots?

Reply:
Great suggestion, just updated the PR! 👍🏻